src/proto/client/mutation.rs

```diff
@@ -0,0 +1,182 @@
+use crate::{
+    proto::single::{MutationAction, MutationType},
+    Client, Error, JobId, MutationFilter, MutationTarget,
+};
+use std::borrow::Borrow;
+
+impl Client {
+    /// Re-enqueue the jobs.
+    ///
+    /// ***Warning!*** The `MUTATE` API is not supposed to be used as part of application logic,
+    /// you will want to use it for administration purposes only.
+    ///
+    /// This method will immediately move the jobs from the targeted set (see [`MutationTarget`])
+    /// to their queues. This will apply to the jobs satisfying the [`filter`](crate::MutationFilter).
+    ///
+    /// ```no_run
+    /// # tokio_test::block_on(async {
+    /// # use faktory::{JobId, Client, MutationTarget, MutationFilter};
+    /// # let mut client = Client::connect().await.unwrap();
+    /// let job_id1 = JobId::new("3sgE_qwtqw1501");
+    /// let job_id2 = JobId::new("3sgE_qwtqw1502");
+    /// let failed_ids = [&job_id1, &job_id2];
+    /// let filter = MutationFilter::builder().jids(failed_ids.as_slice()).build();
+    /// client.requeue(MutationTarget::Retries, &filter).await.unwrap();
+    /// # });
+    /// ```
+    pub async fn requeue<'a, F>(&mut self, target: MutationTarget, filter: F) -> Result<(), Error>
```

Comment:
The argument name `target` makes me think that the jobs will be moved _to_ this, but I don't think that's the case. Isn't it more like `source` or `candidate_job_set` or something?